deploy/template.yaml

```diff
@@ -0,0 +1,569 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: "AWS::Serverless-2016-10-31"
+Description: "Digital Identity IPV CRI F2F API"
+
+Parameters:
+  CodeSigningConfigArn:
+    Type: String
+    Default: "none"
+    Description: >
+      The ARN of the Code Signing Config to use, provided by the deployment pipeline
+  Environment:
+    Description: "The environment type"
+    Type: "String"
+    AllowedValues:
+      - "dev"
+      - "build"
+      - "staging"
+      - "integration"
+      - "production"
+    ConstraintDescription: Ensure environment variable is set to one of dev, build, staging, integration or production.
+  PermissionsBoundary:
+    Description: "The ARN of the permissions boundary to apply when creating IAM roles"
+    Type: String
+    Default: "none"
+  SecretPrefix:
+    Type: String
+    Default: "none"
+    Description: Secrets name prefix
+  VpcStackName:
+    Type: String
+    Default: "vpc-cri"
+    Description: The name of the VPC stack deployed.
+  L2DynamoStackName:
+    Type: String
+    Default: "f2f-cri-ddb"
+    Description: The name of the L2 DynamoDB stack deployed.
+  L2KMSStackName:
+    Type: String
+    Default: "f2f-cri-kms"
+    Description: The name of the L2 DynamoDB stack deployed.
+
+Mappings:
+  EnvironmentConfiguration: # This is where you store per-environment settings.
+    dev:
+      logretentionindays: 3
+      apiTracingEnabled: true
+    build:
+      logretentionindays: 3
+      apiTracingEnabled: true
+    staging:
+      logretentionindays: 3
+      apiTracingEnabled: true
+    integration:
+      logretentionindays: 30
+      apiTracingEnabled: false
+    production:
+      logretentionindays: 30
+      apiTracingEnabled: false
+  EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
+    dev:
+      ISSUER: 'https://review-c.dev.account.gov.uk'
```

Comment:
domain for F2F needs to be `review-o`

Reply:
[Resolved](https://github.com/alphagov/di-ipv-cri-f2f-api/pull/9/commits/2348c502ea82f8c3e1d2fee045d6c6092e6795b9)